zmon_worker_monitor/zmon_worker/tasks/main.py

```diff
@@ -1058,11 +1058,7 @@ def _check_result_limit(self, result):
                 raise ResultSizeError(
                     'Result keys count ({}) exceeded the maximum value: {}'.format(key_count, self.max_result_keys))
 
-        result_str = ''
-        if isinstance(result, dict) or isinstance(result, list):
-            result_str = json.dumps(result).replace(' ', '')
-        elif isinstance(result, basestring):
-            result_str = result
+        result_str = json.dumps(result).replace(' ', '')
```

Comment:
why do we replace spaces? maybe just configure appropriate `json.dumps` separators? https://docs.python.org/3/library/json.html#json.dump